server/render/index.js

```diff
@@ -89,10 +89,12 @@ export function serverRender( req, res ) {
 		context.i18nLocaleScript = '//widgets.wp.com/languages/calypso/' + context.lang + '.js';
 	}
 
+	// only serve ssr content to logged out folks
 	if (
 		config.isEnabled( 'server-side-rendering' ) &&
 		context.layout &&
 		! context.user &&
+		! req.cookies.wordpress_logged_in &&
```

Comment:
Hmm, then again, this should be mostly equivalent to the `! context.user` check on the line above.

Reply:
i added a log line and it appears that there's no context.user, but there is a cookie.... so we're not getting user like we expect? (running on docker)

Reply:
ah. we disabled user bootstrapping on all envs other than prod in https://github.com/Automattic/wp-calypso/pull/17849